chainer/functions/loss/sigmoid_cross_entropy.py

```diff
@@ -67,14 +83,23 @@ def sigmoid_cross_entropy(x, t, use_cudnn=True, normalize=True):
             determines the normalization constant. If true, this function
             normalizes the cross entropy loss across all instances. If else,
             it only normalizes along a batch size.
+        reduce (bool): Variable holding a string value which
+            determines whether to reduce the shape of the input.
+            If it is `mean`, it computes the sum of cross entropy
```

Comment:
Use two backquotes. Single backquotes mean italic in rst. And, use double quotes to represent str.
<pre>``'mean'``</pre>